src/fides/api/ops/api/v1/endpoints/privacy_notice_endpoints.py

```diff
@@ -161,10 +161,12 @@ def get_privacy_notice(
     *,
     privacy_notice_id: str,
     db: Session = Depends(deps.get_db),
+    request: Request,
 ) -> PrivacyNotice:
     """
     Return a single PrivacyNotice
     """
+    unescape = request.headers.get("unescape-safestr")
     return get_privacy_notice_or_error(db, privacy_notice_id)  # type: ignore[return-value]
```

Comment:
At some point we need to call the `unescape` method, but I'm a bit muddied on where that should be. These appear to all operate on the pydantic model, while the `escape`/`unescape` logic is on the sqlalchemy model